### ISSUE TITLE
Instant-NGP fails to run

### ISSUE THREAD

**Issue by tancik** [nerfstudio-project/nerfstudio#201]
When running Instant-NGP it fails at the start of training with the following error:
```
The shape of the mask [1] at index 0 does not match the shape of the indexed tensor [8171, 3] at index 0
  ...
  File "/projects/pyrad/pyrad/engine/trainer.py", line 119, in train
    loss_metric_dict = self.train_iteration(ray_indices, batch, step)
  File "/projects/pyrad/scripts/run_train.py", line 128, in _train
    trainer.train()
  File "/projects/pyrad/scripts/run_train.py", line 161, in launch
    main_func(local_rank=0, world_size=1, config=config)
  File "/projects/pyrad/scripts/run_train.py", line 221, in main
    launch(
  File "/projects/pyrad/scripts/run_train.py", line 232, in <module>
    main()  # pylint: disable=no-value-for-parameter
```

The relevant lines of code:
```
def get_loss_dict(self, outputs, batch):
    device = self.get_device()
    image = batch["image"].to(device)
    if "mask" in outputs:
        mask = outputs["mask"]
        rgb_loss = self.rgb_loss(image[mask], outputs["rgb"][mask])
```
`mask` exists, but it is the wrong shape, in this case it is of size [1] and the value is `false`

This may be related to the recent instant-ngp speedup pr #193
